0.10/README.md

```diff
@@ -90,18 +90,52 @@ file inside your source code repository.
 
 Example: DATABASE_USER=sampleUser
 
-Hot deploy
+Development Mode
 ---------------------
-Hot deploy is not supported in this image. To support it, make sure that modules responsible for the server reload are installed upon the build process. Those modules are:
+This image supports development mode. This mode can be switched on and off with the environment variable `DEV_MODE`. `DEV_MODE` can either be set to `true` or `false`.
+Development mode supports two features:
+* Hot Deploy
+* Debugging
+
+The debug port can be speicifed with the environment variable `DEBUG_PORT`. `DEBUG_PORT` is only valid if `DEV_MODE=true`.
+
+A simple example command for running the docker container in production mode is:
+```
+docker run --env DEV_MODE=true my-image-id
+```
+
+To run the container in development mode with a debug port of 5454, run:
+```
+$ docker run --env DEV_MODE=true DEBUG_PORT=5454 my-image-id
+```
+
+To run the container in production mode, run:
+```
+$ docker run --env DEV_MODE=false my-image-id
+```
+
+By default, `DEV_MODE` is set to `true`, and `DEBUG_PORT` is set to `5858`.
```

Comment:
I'm open to whichever makes sense from the user's perspective. I suppose that since dev_mode is an additional feature of the image, the user should explicitly state that they want to use it. The current functionality of the image is more similar to having dev_mode off, so this would make sense for not breaking current functionality.

So, @bparees, I agree. Should I go ahead and make the necessary changes?